src/schemas/fairdata-ida.ui.js

```diff
@@ -55,6 +55,16 @@ export default {
 			'tabFormat': false,
 		},
 	},
+	'/properties/issued': {
+		'tab': 'description',		
```

Comment:
Was this changed on purpose

Reply:
Yes. It was in the hidden 'extra' tab but it turns out the it's required for PAS stuff so now we show it to the user in the 'description' tab.